enterprise/server/backends/redis_execution_collector/redis_execution_collector.go

```diff
@@ -276,6 +276,12 @@ func (c *collector) DeleteExecutions(ctx context.Context, iid string) error {
 	return c.rdb.Del(ctx, getExecutionKey(iid)).Err()
 }
 
+// SoftDeleteExecutions decreases the TTL of executions data so they're cleaned up soon,
+// but not immediately.
+func (c *collector) SoftDeleteExecutions(ctx context.Context, iid string) error {
```

Comment:
sorry I think I was misusing the term "SoftDelete" :sweat_smile: Seems like it has a very different meaning in the context of databases. Could maybe call it `ExpireExecutions` and pass the TTL as a param